web/themes/custom/usagov/sass/_uswds-theme-custom-styles.scss

```diff
@@ -73,8 +73,7 @@ i.e.
 @media (min-width: 40em) {
   .usa-section {
       padding-bottom: 4rem;
-      padding-top: 3rem;
-  }
```

Comment:
this missing closing bracket is causing an error while compiling the theme


```
Browserslist: caniuse-lite is outdated. Please run:
  npx update-browserslist-db@latest
  Why you should do it regularly: https://github.com/browserslist/update-db#readme
[17:16:54] Finished 'buildSprite' after 177 ms
[17:16:54] Starting 'renameSprite'...
[17:16:54] Finished 'renameSprite' after 2.4 ms
[17:16:54] Starting 'cleanSprite'...
[17:16:54] Finished 'cleanSprite' after 3.64 ms
sass/_uswds-theme-custom-styles.scss
Error: expected "}".
    ╷
955 │ }
    │  ^
    ╵
  sass/_uswds-theme-custom-styles.scss 955:2  @forward
  sass/styles.scss 72:1                       root stylesheet
[17:16:58] Finished 'buildSass' after 3.9 s
[17:16:58] Finished 'update' after 4.99 s
```

Reply:
@scottqueen-bixal thanks Scott, the issue is resolved and ready to approve.

Reply:
this is now compiling